### DISCUSSION TITLE
Error: TypeError: green is not a function

### DISCUSSION THREAD

**Discussion by ttquoccuong** [quasarframework/quasar#13655] [Quasar App Extensions] [answered]
Hi,
I get this error when create a extension.
Full error message:
```
Error: TypeError: green is not a function
    at printMessage (/Users/<user>/.quasar-starter-kits/quasarframework-quasar-starter-kit-ui#master/utils/index.js:106:4)
    at /Users/<user>/.quasar-starter-kits/quasarframework-quasar-starter-kit-ui#master/utils/index.js:159:9
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```
node v16.14.2
quasar cli v1.3.2
Anyone can fix this?

**Comment by suleimanas** [comment]
sorry nothing to tell, no context

**Comment by MilosPaunovic** [comment]
What we could suggest is to remove `node_modules` and `package-lock.json` and do a fresh install of dependencies. That `green` is part of `kolorist` package used, so, potentially, something in `node_modules` got broken.

**Reply by wsteinbrenner** [reply]
This does not help. Trying with empty folder produces:

yarn install v1.22.19
warning package.json: License should be a valid SPDX license expression
info No lockfile found.
warning quasar-ui-yfu-commons-ui@0.0.1: License should be a valid SPDX license expression
[1/4] Resolving packages...
warning @quasar/app > cssnano > cssnano-preset-default > postcss-svgo > svgo > stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
warning cssnano > cssnano-preset-default > postcss-svgo > svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
warning cssnano > cssnano-preset-default > postcss-svgo > svgo > stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning " > @quasar/app@3.3.3" has unmet peer dependency "vue@^3.2.29".
warning " > @quasar/app@3.3.3" has unmet peer dependency "vue-router@^4.0.12".
[4/4] Building fresh packages...
success Saved lockfile.
Done in 30.31s.
Error: TypeError: green is not a function
    at printMessage (/home/wstein/.quasar-starter-kits/quasarframework-quasar-starter-kit-ui#master/utils/index.js:106:4)
    at /home/wstein/.quasar-starter-kits/quasarframework-quasar-starter-kit-ui#master/utils/index.js:159:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

**Comment by yusufkandemir** [comment, marked as answer]
Don't use `quasar create project-name --kit ui`, use `yarn create quasar`/`npm init quasar`, then select UI kit when asked. The old UI kit is broken, but since it's been archived there is no need to fix it, just use the new way.

**Reply by rstoenescu** [reply]
Also, make sure that you have the latest `@quasar/cli` globally installed (v1.3.2)

**Reply by wsteinbrenner** [reply]
THX for reply. Did a yarn add global @quasar/cli. Result still the same:

[*] Installing project dependencies ...

yarn install v1.22.19
warning package.json: License should be a valid SPDX license expression
info No lockfile found.
warning quasar-ui-yfu-commons-ui@0.0.1: License should be a valid SPDX license expression
[1/4] Resolving packages...
warning @quasar/app > cssnano > cssnano-preset-default > postcss-svgo > svgo > stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
warning cssnano > cssnano-preset-default > postcss-svgo > svgo@1.3.2: This SVGO version is no longer supported. Upgrade to v2.x.x.
warning cssnano > cssnano-preset-default > postcss-svgo > svgo > stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning " > @quasar/app@3.3.3" has unmet peer dependency "vue@^3.2.29".
warning " > @quasar/app@3.3.3" has unmet peer dependency "vue-router@^4.0.12".
[4/4] Building fresh packages...
success Saved lockfile.
Done in 30.31s.
Error: TypeError: green is not a function
    at printMessage (/home/wstein/.quasar-starter-kits/quasarframework-quasar-starter-kit-ui#master/utils/index.js:106:4)
    at /home/wstein/.quasar-starter-kits/quasarframework-quasar-starter-kit-ui#master/utils/index.js:159:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

wstein@asgard:~/Workspace/Rails/Customer/Project/vue-common-components-module$ quasar --version
1.3.2

Question: Does the fix below, do the same thing as this kit?

**Reply by rstoenescu** [reply]
`yarn create quasar` -- do not use the deprecated "quasar create" command

**Reply by wsteinbrenner** [reply]
Ah my fault, I've not seen that you you've replied to this comment. So the old command is deprecated for sure?  This was what i wanted to explore because it is still in the official docs (see https://quasar.dev/app-extensions/development-guide/introduction). So I only wanted to check if those calls are equivalent. Sorry for wasting your time but thank you for the confirmation.

**Reply by rstoenescu** [reply]
Yes, it's deprecated. Missed updating docs on that specific page (just pushed a fix for docs).

**Reply by wsteinbrenner** [reply]
Ok so it helped a little bit THX ;)
